statsmodels/tsa/holtwinters.py

```diff
@@ -973,10 +974,13 @@ def _predict(self, h=None, smoothing_level=None, smoothing_slope=None,
         formatted = formatted.loc[included]
 
         hwfit = HoltWintersResults(self, self.params, fittedfcast=fitted,
-                                   _fittedvalues=fitted[:-h - 1], fcastvalues=fitted[-h - 1:],
-                                   sse=sse, level=level, slope=slope, season=season, aic=aic,
+                                   fittedvalues=fitted[:-h - 1],
```

Comment:
AFAIK, this doesn't work
we cannot override a cached property by an attribute

Reply:
In `__init__` this now explicitly sets `self._fittedvalues = fittedvalues`

Reply:
I was skimming to fast.

Reply:
Does this also resolve your comment above in `__init__`?